### PR TITLE
PR for Alert Component

### PR DESCRIPTION
**Pull request by manojasarsa**
Added an alert component.

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    display: flex;
+    flex-direction: column;
+}
+
+.alert_success {
+    display: flex;
+    margin: 1rem;
+    padding: 1rem;
+    color: #1b7052;
+    background-color: #84f1c9;
```
Comment (msdeshmukh009) [inline components/alert/alert.css]: Maybe make variables for these colors as you will be using them again.

**Review by vijaytembugade — APPROVED** [review]
image
I think, You dont have to give this link for each component , although I think you internsions of giving link for each component are good but still, it would be good if you put all together and give that link

File: components/avatar/avatar.html [hunk 2]
```diff
@@ -68,24 +68,31 @@ <h2 class="avatar_head">Avatar</h2>
...
-                style="width: 80vw; height: 30vh; border:0; transform: scale(1); overflow:hidden;"
-                sandbox="allow-scripts allow-same-origin">
-            </iframe>
+            <div class="display_code">
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=material&wt=none&l=css&width=680&ds=true&dsyoff=0px&dsblur=0px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Droid+Sans+Mono&fs=14.5px&lh=147%25&si=false&es=2x&wm=false&code=%253Clink%2520rel%253D%2522stylesheet%2522%2520%250Ahref%253D%2522https%253A%252F%252Fzero-ui.netlify.app%252Fcomponents%252Favatar%252Favatar.css%2522%253E"
+                    style="width: 594px; height: 117px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (msdeshmukh009) [inline components/avatar/avatar.html:74]: You can create a class using these styles.

**Review by msdeshmukh009 — APPROVED** [review]
Rest looks good to me. UI is looking clean and nice!!

File: components/alert/alert.css (hunk 1 above)
Comment (manojasarsa) [reply components/alert/alert.css]: Thanks for reminding, Mahesh. I forget this. Made changes now.

**Pull request merged**